knowledge_graph/make_graph.py

```diff
@@ -188,48 +141,7 @@ def remove_edge_properties_from_nodes(G, to_remove):
             for node in list(G.nodes[node]["properties"][prop])
             if node not in list(to_delete)
         ]
-
-
-def save_graph_to_pickle(G, outfile_path):
```

Comment:
These functions are supposed to enable us to save the knowledge graph in other formats. Would be good to move them all to utils if possible, but I'm not sure if they all work with our current format of the knowledge graph.

Reply:
Can you elaborate? If I'm understanding correctly, it's unclear to me what that has to do with where we locate these functions.

Reply:
To clarify: these functions weren't deleted, just moved to a utils module which make_graph imports from.